cmd/ssl-watch/utils.go

```diff
@@ -1,7 +1,7 @@
 package main
```

Comment:
why do you call this part `utils.go` if that is definitely an `app.go`

Reply:
it started as utils, but eventually became about the app =)